plugins/catalog-backend-module-github/api-report.md

```diff
@@ -253,7 +253,7 @@ export class GithubOrgReaderProcessor implements CatalogProcessor {
 
 // @public
 export type GithubTeam = {
-  id: string;
+  databaseId: number;
```

Comment:
I think I would prefer `id` here tbh. Or does `id` mean something else here? Reading `databaseId` make me think it's something in our postgres database.

I'm open to suggestions perhaps @kissmikijr or @pjungermann has any?

Reply:
I used databaseId here because it's the value comes from the [Github Graphql query](https://github.com/backstage/backstage/pull/14870/files#diff-50740c9b7b864c9aad8c13fe8fb48cdfab8e52f91c8c2cd6ee99cea8c10e7e17R190), I don't know a way to change from the transformation, sorry